CHANGELOG.md~

```diff
@@ -0,0 +1,158 @@
+## [1.0.5] - 2019-12-13
```

Comment:
Is this a temporary file left by the OS? :)

Reply:
Looks like that, let me clean it up :)